mechanicalsoup/browser.py

```diff
@@ -53,10 +53,19 @@ def __init__(self, session=None, soup_config={'features': 'lxml'},
 
         self.soup_config = soup_config or dict()
 
+    @staticmethod
+    def __looks_like_html(response):
+        """Guesses entity type when Content-Type header is missing.
+        Since Content-Type is not strictly required, some servers leave it out.
+        """
+        text = response.text.lstrip()
+        return text.startswith('<html') or text.startswith('<!DOCTYPE')
```

Comment:
This could be case insensitive, since at least `<HTML` is possible too.

Reply:
fixed